conf/defaults.ini

```
@@ -1105,6 +1105,26 @@ min_interval = 10s
 # (concurrent queries per rule disabled).
 max_state_save_concurrency = 1
 
+[unified_alerting.external_alertmanager]
```

```suggestion
[remote.alertmanager]
```

I'd detach this from unified alerting - it's such a niche configuration setting that we might as well not couple it.